common/logging/log4rs-sample.yml

```diff
@@ -32,9 +32,10 @@ appenders:
 
 # Set the default logging level to "error" and attach the "stdout" appender to the root
 root:
-  level: warn
+  level: info
   appenders:
     - base_layer
+    - stdout
```

Comment:
We've got a debug sample logger for this.

The production logger is intentionally keeping stdout clean.
You should be using a log util like lnav on the log files themselves to read logs.

Reply:
The problem is actually that without this, errors are not finding their way to stdout. The program starts logging to stdout, but then after the config is loaded, redirects all logs to the file instead and then ends.
```
$ Tari Base Node
$ Copyright 2019-2020. The Tari Community
$ Version 0.0.5

Press Ctrl-C to quit..
Config file does not exist. Would you like to create a new one (Y/n)?

Installing new config file at /Users/mikethetike/.tari/config.toml
Logging configuration file does not exist. Would you like to create a new one (Y/n)?

Installing new logfile configuration at /Users/mikethetike/.tari/log4rs.yml
Initializing logging according to "/Users/mikethetike/.tari/log4rs.yml"
<program ends>
$ 
```
Took me a while to realise that the error was going to the file.

Reply:
Ok, but then let's fix the config file to do what we want.

The logger must be called `base_node` and not `stdout`. Then it works:

```
$ Tari Base Node
$ Copyright 2019-2020. The Tari Community
$ Version 0.0.5

Press Ctrl-C to quit..
Initializing logging according to "/Users/cayle/.tari/log4rs.yml"
ERROR Node identity information not found. The node identity file, /Users/cayle/.tari/testnet/node_id.json, has an error. j s o n error. You can update the configuration file to point to a valid node identity file, or re-run the node with the --create_id flag to create anew identity.
```

Reply:
Done